phalcon/http/request.zep

```diff
@@ -320,6 +324,15 @@ class Request implements RequestInterface, InjectionAwareInterface
 	}
 
 	/**
+	 * Set request raw body for unit test
+	 */
+	public function setRawBody(string rawBody) -> <Request>
+	{
+	    let this->_rawBody = rawBody;
+	    return this;
+	}
```

Comment:
Please see my unit test case, because Phalcon get request body by `file_get_contents("php://input")`, but we are not able to change `php://input`, which means we are not able to mock a request and make unit test for a request with rawbody. I think add `setRawBody`  is necessary